### ISSUE TITLE
Enforce disitribution mimetype

### ISSUE THREAD

**Issue by coret** [netwerk-digitaal-erfgoed/register#177]
From requirements:

> Therefore, publishers SHOULD add at least one distribution. Each distribution MUST have at least a MIME format and the URL where the distribution can be accessed.

This should be enforced via the SHACL validation

**Comment by ddeboer** [comment]
`schema:encodingFormat` is already required in our SHACL (with severity violation). What else needs to be done here? It seems impossible to validate whether an actual mime type is supplied, expect perhaps requiring `something something / something something`.

**Comment by coret** [comment]
I agree, providing a never ending list of mime types seems not the way to go.
I propose adding a `sh:pattern ^[a-z]+\/[a-z0-9\\-\\+]+$`

**Comment by ddeboer** [comment]
What is the source for that regex? I looks to be missing `.` (and possibly other options), e.g. `application/vnd.openxmlformats-officedocument.spreadsheetml.sheet`.

**Comment by coret** [comment]
Candidate regexps:

`(application|audio|font|example|image|message|model|multipart|text|video|x-(?:[0-9A-Za-z!#$%&'*+.^_`|~-]+))/([0-9A-Za-z!#$%&'*+.^_`|~-]+)`
test via [https://regex101.com/r/NnHsT4/1](https://regex101.com/r/NnHsT4/1)

`\w+\/[-+.\w]+`
test via [https://regex101.com/r/zomRRR/1](https://regex101.com/r/zomRRR/1)
